ReactiveCocoa.xcodeproj/xcshareddata/xcschemes/ReactiveCocoa-Mac.xcscheme

```diff
@@ -3,8 +3,8 @@
    LastUpgradeVersion = "0700"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "YES"
-      buildImplicitDependencies = "YES">
+      parallelizeBuildables = "NO"
+      buildImplicitDependencies = "NO">
```

Comment:
As mentioned, `xctool` requires dependencies to be specified explicitly, meaning there should be no implicit dependencies. Disabling the finding of implicit dependencies should also help avoiding any confusion.

More related (possibly outdated) background:
- http://www.openradar.me/20490378#ag9zfm9wZW5yYWRhci1ocmRyFAsSB0NvbW1lbnQYgICA4LKcjwsM

  > I found that Implicit Dependencies would only mess up if I didn't have all dependencies explicitly configured. I I made sure that all dependencies where explicit, the problem would go away.

- http://pewpewthespells.com/blog/managing_xcode.html#scheme-action

  > However there are some edge-cases to this behavior. If you find yourself in a situation where you cannot rely on Xcode's ability to resolve the necessary target dependencies correctly, then both "Parallelize Build" and "Find Implicit Dependencies" should be disabled on the scheme.

- http://andymolloy.net/workspace-interproject-dependencies-and-xcodebuild/

  > If you have an Xcode workspace containing multiple projects, and a scheme in that workspace depends on targets in more than one of those projects, Xcode’s own automatic dependency detection may well work, but xcodebuild will fail. [...] I also had to uncheck the Parallelize Targets box.